src/sinks/mod.rs

```diff
@@ -65,6 +65,7 @@ pub mod socket;
 pub mod splunk_hec;
 #[cfg(feature = "sinks-statsd")]
 pub mod statsd;
+pub mod tap;
```

Comment:
This is the only sink that doesn't have a feature flag. I initially had an `api` feature here, but that coupling isn't strictly required-- tap could technically be used as a generalised broadcast of log events.